### ISSUE TITLE
the problem about bias_W_fc

### ISSUE THREAD

**Issue by lyyiangang** [czk32611/Gaze_Decomposition#1]
Hi,
Thanks for your open source code. when reading the code,  some code confuse me a lot. when calculate total loss, you use the code below in train.py.
```
    total_loss = (tf.reduce_mean(est_loss) + 1e-3 * l2_loss + 
                  tf.abs(tf.reduce_mean(bias_W_fc[:num_subj,0]))+tf.abs(tf.reduce_mean(bias_W_fc[num_subj:,0]))+
                  tf.abs(tf.reduce_mean(bias_W_fc[:num_subj,1]))+tf.abs(tf.reduce_mean(bias_W_fc[num_subj:,1])))
```
My question is why we can get the bias by using the loss
```
 tf.abs(tf.reduce_mean(bias_W_fc[:num_subj,0]))+tf.abs(tf.reduce_mean(bias_W_fc[num_subj:,0]))+
                  tf.abs(tf.reduce_mean(bias_W_fc[:num_subj,1]))+tf.abs(tf.reduce_mean(bias_W_fc[num_subj:,1])))
```
I think after training, the bias_W_fc could be any number instead of the bias of visual axes and optical axes.

Appreciate your  reply.
Thanks
